src/plugins/wizard/public/application/utils/wizard_state_validation.ts

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import Ajv from 'ajv';
+
+export const wizardStateValidation = (wizardState) => {
```

Comment:
Can we add some unit tests for this?

Reply:
unit tests are added in src/plugins/wizard/public/application/utils/wizard_state_validation.test.ts